src/core/systemd.pc.in

```diff
@@ -21,6 +21,8 @@ systemdsystemunitpath=${systemdsystemconfdir}:/etc/systemd/system:/run/systemd/s
 systemduserunitpath=${systemduserconfdir}:/etc/systemd/user:/run/systemd/user:/usr/local/lib/systemd/user:/usr/local/share/systemd/user:${systemduserunitdir}:/usr/lib/systemd/user:/usr/share/systemd/user
 systemdsystemgeneratordir=${rootprefix}/lib/systemd/system-generators
 systemdusergeneratordir=${prefix}/lib/systemd/user-generators
+systemdsystemgeneratorpath=${systemdsystemgeneratordir}:/etc/systemd/system-generators:/run/systemd/system-generators:/usr/local/lib/systemd/system-generators
+systemdusergeneratorpath=${systemdusergeneratordir}:/etc/systemd/user-generators:/run/systemd/user-generators:/usr/local/lib/systemd/user-generators
```

Comment:
Seems mostly OK, but the order is wrong.
See src/shared/path-lookup.

Reply:
oh, ok. 
I force pushed the correct order.